addons/project/data/mail_template_data.xml

```diff
@@ -7,14 +7,15 @@
             <field name="model_id" ref="project.model_project_task"/>
             <field name="subject">Reception of {{ object.name }}</field>
             <field name="use_default_to" eval="True"/>
-            <field name="description">Set this template on a project's stage to automate email when tasks reach stages</field>
+            <field name="description">Automatically send an email to customers when a task reaches a specific stage in a project by setting this template on that stage</field>
             <field name="body_html" type="html">
 <div>
-    Dear <t t-out="object.partner_id.name or 'customer'">Brandon Freeman</t>,<br/>
-    Thank you for your enquiry.<br />
-    If you have any questions, please let us know.
-    <br/><br/>
-    Thank you,
+    Dear <t t-out="object.partner_id.name or 'customer'">Brandon Freeman</t>,<br/><br/>
+    Thank you for contacting us. We appreciate your interest in our products/services.<br/>
+    Our team is currently reviewing your inquiry and will respond to your email as soon as possible.<br/>
+    If you have any further questions or concerns in the meantime, please do not hesitate to let us know. We are here to help.<br/><br/>
+    Thank you for your patience.<br/>
+    Best regards,
```

Comment:
Woupsie, I completely forgot but we need to update the record in the migration script. 🙂 

Can you add the migration script? 🙂 

Thanks in advance! 🙂

Reply:
Hello @xavierbol 
Done!!
Thanks :innocent: